docs/DeveloperGuide.adoc

```diff
@@ -427,6 +427,102 @@ image::UndoRedoActivityDiagram.png[width="650" align="center"]
 ** Cons: This requires dealing with commands that have already been undone, in which we must remember to skip these commands. This violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things.
 // end::undoredo[]
 
+//tag::Person Panel[]
+=== Person Panel
+The `PersonPanel` replaces the previous `BrowserPanel`, which is called upon using the methods in `CenterPanel`.
+`PersonPanel` will be used to display contact's information, groups and preferences depending on user's actions.
+
+==== Layout Implementation
+The specifications for the layout of PersonPanel is in `PersonPanel.fxml`. The figure below will show the visual layout of the `PersonPanel`.
+
+.Visual layout of PersonPanel
+image::PersonPanelLayout.png[width="800" align="center"]
+
+Notably, the contact's group and preferences are coloured as these are important information to the user.
+We plan to implement more features for v2.0, such as a profile picture for the contact.
+
+==== Current Implementation
+By utilising the `EventsCenter`, PersonPanel is able to display contact's particulars, its
+groups and preferences through selecting its respective `PersonCard`.
```

Comment:
Perhaps you can rephrase “through selecting its respective PersonCard” to “when its respective PersonCard is selected.”